src/commands/install.ts

```diff
@@ -175,21 +175,24 @@ export default class Install extends Command {
       let profileName: string;
 
       if (flags.local) {
+        profileName = trimExtension(
+          profilePathParts[profilePathParts.length - 1]
+        );
```

Comment:
I know this code existed before but this doesn't look very foolproof to me. Maybe we should extract the profile name from the file itself?

Reply:
The whole approach is circular. We cannot 100% know the document id until we parse the document header, but we don't parse the header until we check the cache, and the file names in the cache are derived from the document id...

I suppose this would need a function in the parser that only parses the header from the document?

Reply:
We are extracting that if we are dealing with local files.
I would probably add object `{profile: string, scope?: string, version?: string}` to the whole install logic input and add  scope/name/version check when loading local file. This is a question of how much do we "connect" name of a file and actual content of the file. 🤷

Anyway, I added some tests and I think it's good to go (for now).